api/models/release.go

```diff
@@ -18,7 +19,7 @@ import (
 	"github.com/aws/aws-sdk-go/service/iam"
 	"github.com/convox/rack/api/crypt"
 	"github.com/convox/rack/api/structs"
-	"github.com/convox/rack/manifest"
+	mp "github.com/convox/rack/manifest"
```

Comment:
Had to do this as I wasn't able to refer to `manifest.Port` in my code otherwise. Open on suggestions for a better way to handle this (or better name for it - `mp` stands for manifest package).

I updated other the other references that I found as well.